packages/trader/src/Modules/SmartChart/Components/__tests__/control-widgets.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import ControlWidgets from '../control-widgets';
+
+jest.mock('Modules/SmartChart', () => ({
+    ...jest.requireActual('Modules/SmartChart'),
+    ChartMode: () => <div>MockedChartMode</div>,
+    DrawTools: () => <div>MockedDrawTools</div>,
+    Share: () => <div>MockedShare</div>,
+    StudyLegend: () => <div>MockedStudyLegend</div>,
+    Views: () => <div>MockedViews</div>,
+}));
+
+describe('<ControlWidgets />', () => {
+    const store_config = mockStore({});
+    const renderComponent = ({ mocked_store_props = store_config }) =>
+        render(
+            <StoreProvider store={mocked_store_props}>
+                <ControlWidgets />
+            </StoreProvider>
+        );
+    const mocked_store_props = mockStore({
+        contract_trade: {
+            updateChartType: jest.fn(),
+            updateGranularity: jest.fn(),
+        },
+    });
+
+    it('Should render all mocked widgets', () => {
+        renderComponent({ mocked_store_props });
```

Comment:
Okay, why do you need `mocked_store_props` and `store_config` if you have only one test case? Will it be better to use `mocked_store_props` inside of `renderComponent` directly?